server/src/main/java/com/ca/lsp/cobol/service/CopybookServiceImpl.java

```diff
@@ -115,9 +115,9 @@ private void handleRequiredCopybookEvent(RequiredCopybookEvent event) {
       String cobolFileName = files.getNameFromURI(event.getDocumentUri());
       settingsService
           .getConfiguration(COPYBOOK_RESOLVE.label, cobolFileName, requiredCopybookName)
-          .thenAccept(sendResponse(cobolFileName, requiredCopybookName));
+          .thenAccept(sendResponse(cobolFileName, requiredCopybookName, event.getCopybookProcessingMode()));
     } else {
-      sendResponse(requiredCopybookName, null, null);
+      sendResponse(requiredCopybookName, null, (Path)null);
```

Comment:
What about creating a new method `sendEmtpyResponse` not to use hacks like this?

Reply:
Yes, I will.